### PR TITLE
[MIG][12.0] fieldservice_googlemaps_address_autocompletion

### PR DESCRIPTION
**Pull request by murtuzasaleh**
Field Service - Google Maps Address Auto-completion
=========================================
This module adds address auto-completition to Field Service Locations.
It also adds logic to create and sync PostGIS geometries between 
FSM Locations and FSM Orders associated with that location.

### CODE REVIEW

**Comment by murtuzasaleh** [comment]
@max3903 Can I move fieldservice_geoengine field code here and work based on web_google_maps module?
Because of [Last Work](https://github.com/OCA/field-service/pull/163#issuecomment-478011492)

**Comment by max3903** [comment]
@murtuzasaleh Let's rename this module `fieldservice_googlemaps_address_autocompletion` and only focus on address autocompletion. No code from `fieldservice_geoengine` please.

**Comment by murtuzasaleh** [comment]
> @murtuzasaleh Let's rename this module `fieldservice_googlemaps_address_autocompletion` and only focus on address autocompletion. No code from `fieldservice_geoengine` please.

Done

**Comment by murtuzasaleh** [comment]
@max3903 @osi-scampbell Can you please review this PR.

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+odoo-addons https://github.com/gityopie/odoo-addons.git 12.0
+
```
Comment (max3903) [inline oca_dependencies.txt]: Please remove this empty line.

Comment (murtuzasaleh) [reply oca_dependencies.txt]: Done

**Review by max3903 — APPROVED** [review]

**Review by osi-scampbell — APPROVED** [review]

**Pull request merged**